libs/blocks/global-navigation/utilities/menu/menu.css

```diff
@@ -5,8 +5,7 @@
 
 .feds-navLink--hoverCaret[aria-expanded = "true"] + .feds-popup,
 .feds-menu-headline[aria-expanded = "true"] + .feds-menu-items {
-  display: flex;
-  flex-direction: column;
+  display: inline-block;
```

Comment:
This will cause issues in the footer

<img width="428" alt="Screenshot 2024-06-17 at 12 09 57" src="https://github.com/adobecom/milo/assets/11267498/27f36477-66cf-45c4-88f6-f0a3bee38f21">

Reply:
Fixed.
Replaced with `display: block;`.